scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -61,11 +61,12 @@
                       },
                       {% for track in sample_tracks %}
                       {
-                        name: '{{ track.name }}',
-                        url: '{{ url_for("alignviewers.remote_static", file=track.url) }}',
-                        indexURL: '{{ url_for("alignviewers.remote_static", file=track.indexURL) }}',
-                        sourceType: 'file',
+                        name: "{{ track.name }}",
                         type: "alignment",
+                        url: "{{ url_for('alignviewers.remote_static', file=track.url) }}",
+                        indexURL: "{{ url_for('alignviewers.remote_static', file=track.indexURL) }}",
+                        sourceType: "file",
+                        format: "{{ track.format }}",
```

Comment:
That one line would have been enough, right? Asking for a friend who will have to merge conflicts.. 😆

Reply:
As you know well, that's the line that does the magic! ;)